src/content/docs/ko/reference/api-reference.mdx

```diff
@@ -370,11 +370,17 @@ true인 경우 쿠키는 https 사이트에만 설정됩니다.
 
 ### `Astro.redirect()`
 
-다른 페이지로 리디렉션하며, 선택적으로 두 번째 매개변수로 [HTTP 응답 상태 코드](https://developer.mozilla.org/ko/docs/Web/HTTP/Status)를 제공할 수 있습니다.
+**타입:** `(path: string, status?: number) => Response`
+
+다른 페이지로 리디렉션할 수 있으며, 선택적으로 [HTTP 응답 상태 코드](https://developer.mozilla.org/ko/docs/Web/HTTP/Status#%EB%A6%AC%EB%8B%A4%EC%9D%B4%EB%A0%89%EC%85%98_%EB%A9%94%EC%8B%9C%EC%A7%80)를 두 번째 매개변수로 제공할 수 있습니다.
 
 페이지 (하위 컴포넌트는 아님)에 리디렉션이 발생하려면 `Astro.redirect()` 결과를 `return`해야 합니다.
 
-다음 예시에서는 기본 HTTP 응답 상태 코드 302를 사용하여 사용자를 로그인 페이지로 리디렉션합니다.
+정적으로 생성된 사이트의 경우 [`<meta http-equiv="refresh">` 태그](https://developer.mozilla.org/ko/docs/Web/HTML/Element/meta#http-equiv)를 사용하여 클라이언트 리디렉션을 생성하며 상태 코드를 지원하지 않습니다.
+
+주문형 렌더링 모드를 사용할 경우 상태 코드가 지원됩니다. Astro는 다른 코드를 지정하지 않는 한 리디렉션된 요청을 기본 HTTP 응답 상태인 `302`로 처리합니다.
+
+다음 예시는 사용자를 로그인 페이지로 리디렉션합니다.
```

Comment:
Awesome job @jsparkdev, I left you a tiny suggestion 😁
```suggestion
다음 예시는 사용자를 로그인 페이지로 리디렉션합니다:
```

Reply:
Thank you!